channels/il.m3u

```diff
@@ -1,51 +1,31 @@
+#EXTINF:-1 tvg-id="Channel 20-IS" tvg-name="IL: ISRAEL Channel 20" tvg-language="Hebrew" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/5/53/Arutz_20.svg/1920px-Arutz_20.svg.png" group-title="עידן+",Channel 20 ערוץ
 http://live.ch20-cdnwiz.com/ch20-abr/ch20-abr/playlist.m3u8
-#EXTINF:-1 tvg-id="Hidabroot Channel-IS" tvg-name="Hidabroot Channel-IS" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/wKi97Pg.png" group-title="",Hidabroot
+#EXTINF:-1 tvg-id="Hidabroot Channel-IS" tvg-name="Hidabroot Channel-IS" tvg-language="Hebrew" tvg-logo="https://upload.wikimedia.org/wikipedia/he/7/7a/Hidabroot_logo.png" group-title="עידן+",Hidabroot
 http://stream5.shidur.net/htvlive2/_definst_/smil:live2.smil/chunklist.m3u8
-#EXTINF:-1 tvg-id="Hidabroot Channel-IS" tvg-name="Hidabroot Channel-IS" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/wKi97Pg.png" group-title="",Hidabroot
-https://stream71.shidur.net/htvlive2/_definst_/smil:live2.smil/chunklist_b1248000.m3u8
-#EXTINF:-1 tvg-id="Hidabroot Channel-IS" tvg-name="Hidabroot Channel-IS" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/wKi97Pg.png" group-title="",Hidabroot
-https://stream72.shidur.net/htvlive2/_definst_/smil:live2.smil/chunklist_b1248000.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/RSssxcw.jpg" group-title="News",i24 News
-https://i24alive1-a.akamaihd.net/6e3dd61ac4c34d6f8fb9698b565b9f50/eu-central-1/5377161796001/profile_0/chunklist_dvr.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/RSssxcw.jpg" group-title="News",i24 News
-https://i24alive1-a.akamaihd.net/6e3dd61ac4c34d6f8fb9698b565b9f50/eu-central-1/5377161796001/profile_0/chunklist_dvr.m3u8?fluxustv.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/8R1UWRw.jpg" group-title="",Kabbalah for the People (Israel)
-http://edge1.il.kab.tv/rtplive/tv66-heb-high.stream/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/8R1UWRw.jpg" group-title="",Kabbalah for the People (Israel)
-https://edge2.uk.kab.tv/rtplive/tv66-heb-medium.stream/playlist.m3u8
-#EXTINF:-1 tvg-id="Kan 11-IS" tvg-name="IL: Kan 11" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/vqfNbEU.png" group-title="",Kan 11
-https://kanlivep2event-i.akamaihd.net/hls/live/747610/747610/source1_2.5k/chunklist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/zCZH65G.png" group-title="Education",Kan Educational
-https://kanlivep2event-i.akamaihd.net/hls/live/747600/747600/source1_2.5k/chunklist.m3u8
-#EXTINF:-1 tvg-id="Knesset Channel-IS" tvg-name="Knesset Channel-IS" tvg-language="Hebrew" tvg-logo="https://i.imgur.com/clKv6Ar.png" group-title="",Knesset Channel
-http://w3.013.gostreaming.tv:1935/Knesset/myStream/playlist.m3u8
```

Comment:
All this links work for me (tested in VLC player)

Reply:
Sorry, but I still don't see these links in the updated version.

Reply:
Kabbalah - it's not a serious TV channel, but I can add it back (there are many such nonsense 'channels' you can find on the internet). Please lmk
Kan11, Kan Educational - they're there. In the previous version there were a number of different streams at different resolutions - which I think is uninteresting. I added just the highest resolution (FHD). (I try to avoid clutter)
Knesset is there (the last one in the list), different URL which I think is more precise (both work, though).

Reply:
Yes, please bring them back.